app/models.py

```diff
@@ -0,0 +1,78 @@
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.dialects.postgresql import UUID, ARRAY
+from sqlalchemy.sql import text
+
+db = SQLAlchemy()
+
+class Base(db.Model):
+    __abstract__ = True  # Indicates that this class should not be created as a table
+    id = db.Column(UUID(as_uuid=True), primary_key=True,
+                   server_default=text("gen_random_uuid()"))
+
+
+class HarvestSource(Base):
+    __tablename__ = 'harvest_source'
+    
+    name = db.Column(db.String, nullable=False)
+    notification_emails = db.Column(ARRAY(db.String))
+    organization_id = db.Column(db.String, nullable=False)
+    frequency = db.Column(db.String, nullable=False)
+    url = db.Column(db.String, nullable=False)
+    schema_type = db.Column(db.String, nullable=False)
+    source_type = db.Column(db.String, nullable=False)
+    harvest_source_id = db.Column(db.String)
+    harvest_source_name = db.Column(db.String)
```

Comment:
what is the difference between id/harvest_source_id and name/harvest_source_name ?

Reply:
@btylerburton we had this discussion one time in the meeting, and added this two columns https://github.com/GSA/data.gov/issues/4612#issuecomment-1960391070, do we still need them?

Reply:
after discussion, we decide to remove these two columns.